website/docs/getting-started/installation.md

```diff
@@ -21,7 +21,7 @@ Tip: If you get an error with the following `npm` commands such as `npx: command
 
 :::tip
 
-For ESM configuration, please see more in details with [ESM guide](../guides/esm-support).
+For ESM configuration, please see more in details with [ESM guide](../../guides/esm-support).
```

Comment:
I didn't see the issue on the doc site

Reply:
Oh I tested again, and like you said, there was no issue, had to look through my history to see that when I got the error, I was on https://kulshekhar.github.io/ts-jest/docs/getting-started/installation/  (<-- Note the trailing slash), where I ended up because i googled "ts-jest getting started" and the google link has the trailing slash.

I imagine there could be some configuration to always/never have trailing slash for consistent linking.

Reply:
Not sure if this option is related https://docusaurus.io/docs/api/docusaurus-config#trailingSlash